asmrun/backtrace_prim.c

```diff
@@ -67,6 +67,14 @@ frame_descr * caml_next_frame_descriptor(uintnat * pc, char ** sp)
   }
 }
 
+int caml_alloc_backtrace_buffer(){
+  Assert(caml_backtrace_pos == 0);
```

Comment:
This seems wrong. When restoring a backtrace, you set caml_backtrace_pos first then allocate a buffer.
This assertion will fail when using reraise_raw_backtrace in debug mode!